docs/reference/optimization/benchmarks.md

```diff
@@ -114,9 +114,7 @@ Using pipelining results in a significant increase in performance.
 
 ### Pitfalls and misconceptions
 
-The first point is obvious: the golden rule of a useful benchmark is to
-only compare apples and apples. Different versions of Redis can be compared
-on the same workload for instance. Or the same version of Redis, but with
+You can compare different versions of Redis on the same workload or the same version of Redis, but with
```

Comment:
I like Salvatore's style of writing. It has a personality to it. I'd appreciate if we can retain some of it.